src/pages/drive/addIns/driveAddIns.tsx

```diff
@@ -0,0 +1,529 @@
+        language="json"
+        code={`"items": [{
+    "version": "1.0",
+    "url": "addinFile.html",
+    "path": "DriveAppLink/",
+    "menuName": {
+        "en": "English Label",
+        "fr": "French Label"
+    }
+}]`}></CodeSample >
+    <InformationalBox>
+        <p>The array of <strong>items</strong> also allows you to have one link item in MyGeotab, and another link item to Geotab Drive</p>
+    </InformationalBox>
+    <p>There are also 2 additional properties that are optional for the configuration file that control the availability to additional page lifecycle methods, <em>onStartup</em> and <em>onShutdown</em>. These configuration properties are boolean, they can be set to <em>true</em> if they are to be used and <em>false</em> when not in use. By default these 2 properties will be set to false if not included in the configuration file.</p>
+    <ol>
+        <li><p><strong>onStartup:</strong> Startup Add-Ins are executed when a driver logs in to the Drive App for the first time.</p></li>
+        <li><p><strong>onShutdown:</strong> The onShutdown property must be set to true to execute an Add-In when logging out of Geotab Drive.</p></li>
+    </ol>
+    <InformationalBox>
+        <p>More information about these 2 methods can be found in this {" "}
+            <a href="https://docs.google.com/document/d/1-r9o9epj61WMmGxRveA9SXR86lQGHcxgMh8lsVXGL54/edit?usp=sharing" target="_blank"
+                rel="noopener noreferrer"
+                aria-label="">document</a>
+            .</p>
+    </InformationalBox>
+
+    <p>e.g.</p>
+    <CodeSample
+        language="json"
+        code={`{
+    "name": "addin-name",
+    "supportEmail": "mysupport@support.com",
+    "version": "1.0.0",
+    "items": [{
+        "url": "index.html",
+        "path": "DriveAppLink/",
+        "menuName": {
+            "en": "Add-in"
+        },
+        "icon": "images/icon.svg"
+    }, {
+        "url": "myGeotabConfigurationPage.html",
+        "path": "AdministrationLink/",
+        "menuName": {
+            "en": "Add-in Configurations"
+        },
+        "icon": "images/icon.svg"
+    }],
+    "onStartup": true,
+    "onShutdown": true
+}`}></CodeSample >
+</div >;
+
+
+const driveLifeCycle: ReactNode = <div className="paragraph">
+    <p>The onStartup and onShutdown properties respectively enable the <strong>startup</strong> and <strong>shutdown</strong> lifecycle methods.</p>
+    <h2>Startup</h2>
+    <p>When the dashboard page is visible, the startup method is only called once. If the user navigates away from the page then navigates back, the startup method is not called again. If the Add-In requires re-initialization, the user must either log out and log in again, or refresh the application.</p>
+    <CodeSample
+        language="javascript"
+        code={`startup: function (freshApi, freshState, initializeCallback) {
+    // Code that needs to be executed on dashboard should go here
+    initializeCallback();
+}`}></CodeSample >
```

Comment:
```suggestion
}`}/>
```
This can be self closing cause it has no children inside of it.